### ISSUE TITLE
Narrator screen reader is not announcing "Please check your email to reset your password" status message: A11y_ASP.NET_Login_ForgetYourPassword_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#56294]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_ASP.NET_Web_Jun2024; #.NETCore; #WCAG4.1.3; #DesktopWeb; #Win11; #FTP; #NVDA; #Narrator; #ChromiumEdge; #Chrome;

### Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22621.3085
Screen readers: Narrator, NVDA 2023.3.3

### Repro Steps:

1. Start Narrator/ NVDA screen reader.
2. Download the "BlazorWebApp" zip file and run .exe file.
3. Command prompt will open, TAB to "http://localhost:5000/" link and press CTRL + ENTER key.
4. TAB to "Login" tab and hit ENTER key. TAB to "Forget your password" link and hit ENTER key.
5. TAB to "Reset password" button and hit ENTER key.
6. Observe the issue.

### Actual Result:
Narrator screen reader is not announcing "Please check your email to reset your password" status message, it is only announcing "Forgot password confirmation" and not announcing whole message.

**Similar issue is also observed with Chrome + NVDA.**

**Similar issue is observed in "BlazorWebAppUnchanged" template.**

### Expected Result:
Narrator screen reader should be announcing "Forgot password confirmation, Please check your email to reset your password" status message.

### User Impact:
Screen reader users will not get complete information about the status appearing on the screen and will miss information about it.

### Attachment
[56294_A11y_ASP.NET_Login_ForgetYourPassword_NVDA.webm](https://github.com/dotnet/aspnetcore/assets/90901519/342e9f64-14d9-4fb1-8255-41afa7fb680a)

[56294_A11y_ASP.NET_Login_ForgetYourPassword_Narrator.webm](https://github.com/dotnet/aspnetcore/assets/90901519/00714a9a-c295-408c-be25-ef184409c67d)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-20-24;

Verified the bug in environment http://localhost:5000/Account/ForgotPasswordConfirmation
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment


![image](https://github.com/user-attachments/assets/c3597d2c-4db8-44ae-8dac-c7c9b00c62ad)

![image](https://github.com/user-attachments/assets/6f8359f1-d6a0-4e68-a38f-3dba9e625228)
